spec.md

```diff
@@ -262,22 +262,22 @@ A digest can be verified by independently calculating `D` and comparing it with
 
 #### Digest Header
 
-To provide verification of http content, any response may include a `Docker-Content-Digest` header.
+To provide verification of http content, any response MAY include a `Docker-Content-Digest` header.
 This will include the digest of the target entity returned in the response.
 For blobs, this is the entire blob content.
 For manifests, this is the manifest body without the signature content, also known as the JWS payload.
-Note that the commonly used canonicalization for digest calculation may be dependent on the mediatype of the content, such as with manifests.
+Note that the commonly used canonicalization for digest calculation MAY be dependent on the mediatype of the content, such as with manifests.
 
-The client may choose to ignore the header or may verify it to ensure content integrity and transport security.
+The client MAY choose to ignore the header or MAY verify it to ensure content integrity and transport security.
 This is most important when fetching by a digest.
-To ensure security, the content should be verified against the digest used to fetch the content.
-At times, the returned digest may differ from that used to initiate a request.
+To ensure security, the content SHOULD be verified against the digest used to fetch the content.
+At times, the returned digest MAY differ from that used to initiate a request.
 Such digests are considered to be from different _domains_, meaning they have different values for _algorithm_.
-In such a case, the client may choose to verify the digests in both domains or ignore the server's digest.
-To maintain security, the client _must_ always verify the content against the _digest_ used to fetch the content.
+In such a case, the client MAY choose to verify the digests in both domains or ignore the server's digest.
+To maintain security, the client _MUST_ always verify the content against the _digest_ used to fetch the content.
```

Comment:
`/s/_MUST_/MUST/`

Italicizing had merit when it was lower case.. but we SHOULD be consistent with the legal spec terms.